auditor/src/routes/advanced_record_filters.rs

```diff
@@ -0,0 +1,193 @@
+// Copyright 2021-2022 AUDITOR developers
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+use crate::domain::{Record, RecordDatabase};
+use chrono::{DateTime, Utc};
+use core::fmt::Debug;
+use serde_qs::actix::QsQuery;
+use sqlx;
+use sqlx::{PgPool, QueryBuilder, Row};
+
+#[derive(thiserror::Error)]
+pub enum GetFilterError {
+    #[error(transparent)]
+    UnexpectedError(#[from] anyhow::Error),
+}
+
+debug_for_error!(GetFilterError);
+responseerror_for_error!(GetFilterError, UnexpectedError => INTERNAL_SERVER_ERROR;);
+
+#[derive(serde::Deserialize, Debug, Clone)]
+pub struct Filters {
+    pub start_time: Option<Operator<DateTime<Utc>>>,
+    pub stop_time: Option<Operator<DateTime<Utc>>>,
+    pub runtime: Option<Operator<String>>,
+}
+
+#[derive(serde::Deserialize, Debug, Clone)]
+pub struct Operator<T> {
+    pub gt: Option<T>,
+    pub lt: Option<T>,
+    pub gte: Option<T>,
+    pub lte: Option<T>,
+}
+
+#[tracing::instrument(name = "Get all records since a given timepoint", skip(filters, pool))]
```

Comment:
The name for the tracing instrument doesn't match with what the function is doing.